snapcraft/cli/__main__.py

```diff
@@ -21,6 +21,7 @@
 import subprocess
 
 from ._runner import run
+from .snapcraftctl._runner import run as run_snapcraftctl  # noqa
```

Comment:
why is this needed here?

Reply:
oh, never mind, console_scripts

Reply:
Yeah, didn't want to duplicate the whole "set UTF-8" thing.